src/boundingboxes.jl

```diff
@@ -46,6 +46,15 @@ function boundingbox(s::Sphere{Dim,T}) where {Dim,T}
   Box(c - r⃗, c + r⃗)
 end
 
+function boundingbox(p::ParaboloidSurface{T}) where {T}
```

Comment:
Try to use the parameter T in the expressions. This will ensure that people can pass unitful coordinates in the future. Also, try to avoid accessing internal fields of the paralaboid with `p.field` syntax. Instead, use the public functions defined by the primitive.